api/game.py

```diff
@@ -232,6 +293,28 @@ def get_player_info(player_names):
             good_roles_in_game.append("Tristan")
             good_roles_in_game.append("Iseult")
 
+    # if we took no lovers and not the older sibling:
+    # we must replace one role with a new role
+    # if there are not enough roles to replace, we remove the older sibling and one other role,
+    # and add the lovers
+    if (
+        sum(gr in ["Tristan", "Iseult"] for gr in good_roles_in_game) == 0
+        and "Older Sibling" not in good_roles_in_game
+        and num_good > 1
+    ):
+        available_roles = (
+            set(good_roles) - set(good_roles_in_game) - set(["Tristan", "Iseult", "Older Sibling"])
+        )
+        if len(available_roles) == 0:
+            good_roles_in_game.remove("Older Sibling")
+            good_roles_in_game.append("Tristan")
+            good_roles_in_game.append("Iseult")
+        else:
+            rerolled = random.choice(good_roles_in_game)
+            good_roles_in_game.remove(rerolled)
+            good_roles_in_game.append(random.sample(set(available_roles), 1)[0])
```

Comment:
I thought this branch was supposed to replace a good roll with the older sibling?